kodilla-rps/src/main/java/com/kodilla/rps/RpsGame.java

```diff
@@ -0,0 +1,74 @@
+package com.kodilla.rps;
+
+import com.kodilla.rps.round.RpsRound;
+import com.kodilla.rps.round.RpsRoundResult;
+
+import java.util.Scanner;
+
+public class RpsGame {
+
+    private RpsGameDefinition rpsGameDefinition;
+    private static boolean end;
+    private static int playerWins;
+    private static int computerWins;
+    private int roundCounter;
+
+    public RpsGame(RpsGameDefinition rpsGameDefinition) {
+        this.rpsGameDefinition = rpsGameDefinition;
+        end = false;
+        playerWins = 0;
+        computerWins = 0;
+        roundCounter = 0;
+    }
+    private void resolveRound(RpsRoundResult result) {
+        switch (result) {
+            case WIN:
+                playerWins++;
+                break;
+            case LOSS:
+                computerWins++;
+                break;
+            case DRAW:
+                break;
+            case REPLAY:
+                replayGame();
+                break;
+            case END:
+                endGame();
+                break;
+            case BAD:
+                break;
```

Comment:
może coś wydrukować na ekranie? Zły ruch i poprosić o nowy ruch w takiej sytuacji (czyli dopóki użytkownik nie wpisze dobrego ruchu, to w ogóle runda nie zwróci nic. A więc wtedy tutaj byśmy w ogóle nie mieli wyniku typu BAD.